setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="serialmfg",
-    version="0.2",
+    version="1.0",
```

Comment:
Undecided on what we should do for version here, but since it's a step change we should probably do a new major version.

Reply:
Agreed. I think 1.0 makes sense for this release. HUGE!